Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -633,7 +633,7 @@ $body = @'
 '@
 
 Write-Host "Registering Fabric.Authorization Client."
-$authorizationClientSecret = Add-ClientRegistration -authUrl $identityServerUrl -body $body -accessToken $accessToken
+$authorizationClientSecret = ([string](Add-ClientRegistration -authUrl $identityServerUrl -body $body -accessToken $accessToken)).Trim()
```

Comment:
Do we also need to cast to a string and then trim when we are adding the Api registration above in line 617?